lib/rspec/hive/exponential_backoff.rb

```diff
@@ -0,0 +1,15 @@
+require 'retryable'
+
+module RSpec
+  module Hive
+    class ExponentialBackoff
+      class << self
+        def retryable(tries: 5, on:)
```

Comment:
A good (and short) read about the upsides of using keyword params https://robots.thoughtbot.com/ruby-2-keyword-arguments